lib/loggly/client.js

```diff
@@ -185,10 +185,12 @@ Loggly.prototype.log = function (msg, tags, callback) {
 
   common.loggly(logOptions, callback, function (res, body) {
     try {
-      var result = JSON.parse(body);
-      self.emit('log', result);
-      if (callback) {
-        callback(null, result);
+      if(body && res.statusCode.toString() === '200'){
```

Comment:
We should check for non 200 code and log the message.

Reply:
@mchaudhary, Here if the returned response code is 200 then the **body** contains **{"response" : "ok"}** which then get parsed normally. Now If the returned response code is other than 200 and in HTML format as I created a scenario in which I was getting status code **503 "Service Unavailable"** then the library breaks to parse the HTML format. 

I am not able to check with other server error codes like 500, 501 and 504 etc to see in what format they are returning the response and other error codes in 400 series like I tested with 403(wrong token) and 404(Not found) but in both the cases the control did not come to parse the body object and there was nothing in the body. 

In the else part, I am also printing the error code with the error message.